lib/puppet/type/file/mode.rb

```diff
@@ -19,16 +19,22 @@ module Puppet
       you can leave `mode` unmanaged and use
       [the puppetlabs/acl module.](https://forge.puppetlabs.com/puppetlabs/acl)
 
-      Numeric modes should use the standard four-digit octal notation of
-      `<setuid/setgid/sticky><owner><group><other>` (e.g. 0644). Each of the
-      "owner," "group," and "other" digits should be a sum of the
-      permissions for that class of users, where read = 4, write = 2, and
-      execute/search = 1. When setting numeric permissions for
-      directories, Puppet sets the search permission wherever the read
-      permission is set.
+      Numeric modes should use the standard octal notation of
+      `<SETUID/SETGID/STICKY><OWNER><GROUP><OTHER>` (e.g. '0644').
```

Comment:
Nah. There was disagreement over whether using specifically _four_ digits should be considered "standard" (answer: talk to your local style guide), but the order in which those digits go is really the standard numerical way to express permissions.